examples/oauth/src/main.rs

```diff
@@ -30,7 +31,7 @@ use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
 static COOKIE_NAME: &str = "SESSION";
 
 #[tokio::main]
-async fn main() {
+async fn main() -> Result<(), AppError> {
```

Comment:
I still think we should `unwrap` the results in main and not return a `Result` at all.

```suggestion
async fn main() {
```